test/pytest_coreneuron/test_basic.py

```diff
@@ -263,7 +263,15 @@ def test_deleted_sec():
             # for valid section first.
             words = str(m).split()
             print("m is " + words[4] + "." + words[2])
-            expect_err("m()")
+            if "name" != words[2]:
+                expect_err("m()")
+
+    print(s)
+    print(seg)
+    print(mech)
+    print(str(s))
+    print(str(seg))
+    print(str(mech))
```

Comment:
most likely don't have full context but printing these to stdout is required?

Reply:
One could capture the output:
https://docs.pytest.org/en/7.4.x/how-to/capture-stdout-stderr.html#accessing-captured-output-from-a-test-function

Then one can check that it prints something; and I think one wont pollute the output of `pytest -s` as much. Probably, advisible to move the `print` tests to their own function.

Here's the idea in use:
https://stackoverflow.com/a/20507769

Reply:
Ah ok. I don't think I have a better suggestion! But a comment like what you said  would be helpful for future readers :)

Reply:
Looks like there are already assert statements for the use of str. So that leaves only the three direct print.  What do you think about leaving those in?

Reply:
I've added a comment indicating the purpose of the prints